server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,409 @@
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 8, 8, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertFStreetCommunityCenterData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+    $reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The F Street Community Center data has already been inserted');
+	}
+	
+	try {
+		$DB_CONN->beginTransaction();
+
+        $siteId = 8; // Manually obtained from PROD DB
+
+		// Tuesdays
+        $dates = getWeeklyDatesFromRange('2020-02-04', '2020-04-07');
+        
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 19:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting F Street Community Center data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertSoutheastCommunityCollegeData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Southeast Community College Data has already been inserted');
+	}
+ 	
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = insertSite('Southeast Community College', '8800 O Street, #T-102, Lincoln, NE 68520', '402-472-9638', FALSE, FALSE);
+
+        // Thursdays, closed March 26th, January 30th not on the flyer
+        $dates = array('2020-02-06', '2020-02-13', '2020-02-20', '2020-02-27', '2020-03-05', '2020-03-12', '2020-03-19', '2020-04-02', '2020-04-09');
+        
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 16:30:00", "$date 18:30:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 16:30:00", 4, 4, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 17:30:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Saturdays, closed March 21st and 28th
+        $dates = array('2020-02-01', '2020-02-08', '2020-02-15', '2020-02-22', '2020-02-29', '2020-03-07', '2020-03-14', '2020-04-04', '2020-04-11');

```

Comment:
The flier doesn't have the Apr 11 date on it, but the paramer sheet does. We need to verify this with Katie.

Reply:
Changed. Not sure why I trusted the spreadsheet here. Sorry!

**For the Southeast Community College site, the site schedule flier says that its last Saturday date is Apr 4, but the excel sheet said Apr 11. Which is the correct last Saturday date for the Southeast Community college site? APR 4**